src/main/java/network/brightspots/rcv/CastVoteRecord.java

```diff
@@ -61,22 +60,23 @@
   // a set is used to handle overvotes
   SortedMap<Integer, Set<String>> rankToCandidateIds;
   // contest associated with this CVR
-  private Integer contestId;
+  private String contestId;
   // tabulatorId parsed from Dominion CVR data
   private Integer tabulatorId;
   // batchId parsed from Dominion CVR data
-  private Integer batchId;
+  private String batchId;
```

Comment:
If we're changing these two to `Strings`, do you think we should do it with the other two IDs as well?